devtools/create-gossipstore.c

```diff
@@ -69,7 +69,7 @@ int main(int argc, char *argv[])
 	struct amount_sat sat;
 	bool verbose = false;
 	char *infile = NULL, *outfile = NULL, *csvfile = NULL, *csat = NULL;
-	int infd, outfd, scidi = 0, channels = 0, nodes = 0, updates = 0;
+	int infd = 0, outfd, scidi = 0, channels = 0, nodes = 0, updates = 0;
```

Comment:
Is the intent that if this program is not given `--input`, it acts as a filter program?

If so, we should probably use `infd = STDIN_FILENO` instead.

Or better yet change this:

    if (infile) {
        infd = open(infile, O_RDONLY);
        if (infd < 0)
            err(1, "opening %s", infile);
    } else
        infd = STDIN_FILENO;


For symmetry with `outfd`.